test/Tester/TestUtils.cs

```diff
@@ -114,20 +114,12 @@ public static class RequestContextTestUtils
     {
         public static void SetActivityId(Guid id)
         {
-#if NETSTANDARD
             RequestContext.ActivityId.Value = id;
```

Comment:
Yeah, the reason it was extracted into separate variable was to [ save us from more conditional compilation statements.](https://github.com/dotnet/orleans/pull/2200#discussion_r80574713).